4-WebApp-your-API/4-1-MyOrg/README.md

```diff
@@ -1,225 +1,269 @@
+        
+   1. Select **ID tokens (used for implicit and hybrid flows)** checkbox.
+
+      ![Configure Web Screen](./ReadmeFiles/ConfigureWebScreen.png)
+
+   1. Click **Save** to save your changes.
+   
+   1. In the app's registration screen, select the **Certificates & secrets** blade in the left to open the page where you can generate secrets and upload certificates.
+   1. In the **Client secrets** section, select **New client secret**:
+      - Type a key description (for instance `app secret`),
+      - Select one of the available key durations (**6 months**, **12 months** or **Custom**) as per your security posture.
+      - The generated key value will be displayed when you select the **Add** button. Copy and save the generated value for use in later steps.
+      - You'll need this key later in your code's configuration files. This key value will not be displayed again, and is not retrievable by any other means, so make sure to note it from the Azure portal before navigating to any other screen or blade.
+          > :bulb: For enhanced security, consider [using certificates](https://github.com/AzureAD/microsoft-identity-web/wiki/Certificates) instead of client secrets.
+         
+   1. In the app's registration screen, select the **API permissions** blade in the left to open the page where we add access to the APIs that your application needs.
+      - Select the **Add a permission** button and then,
+      - Ensure that the **My APIs** tab is selected.
+      - In the list of APIs, select the API `TodoListService-aspnetcore-webapi`.
+      - In the **Delegated permissions** section, select the **Access 'TodoListService-aspnetcore-webapi'** in the list. Use the search box if necessary.
+      - Select the **Add permissions** button at the bottom.
+   
+   #### Configure the client app (TodoListClient-aspnetcore-webapi) to use your app registration
+
+   Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+   > In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+   
+   1. Open the `Client\appsettings.json` file.
+      1. Find the key `Domain` and replace the existing value with your Azure AD tenant name.
+      1. Find the key `TenantId` and replace the existing value with your Azure AD tenant ID.
+      1. Find the key `ClientId` and replace the existing value with the application ID (clientId) of `TodoListClient-aspnetcore-webapi` app copied from the Azure portal.
+      1. Find the key `ClientSecret` and replace the existing value with $pwdCredential.SecretText.
+      1. Find the key `TodoListScope` and replace the existing value with Scope.
+      1. Find the key `TodoListBaseAddress` and replace the existing value with Web.
+            
+  **For more information, visit** [Register Application AAD](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
```

Comment:
>**For more information, visit** [Register Application AAD](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) [](http://example.com/codeflow?start=2&length=147)

good :)

Reply:
I've moved it to the beginning of the chapter